frontend/package.json

```diff
@@ -30,14 +30,16 @@
     "glamor": "^2.20.40",
     "jest": "^22.4.2",
     "prop-types": "^15.6.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "~16.8.4",
+    "react-dom": "~16.8.4",
     "react-helmet": "^5.2.0",
-    "react-transition-group": "^2.2.1"
+    "react-transition-group": "^2.2.1",
+    "react-hot-loader": "^3.1.3",
+    "react-transform-catch-errors": "^1.0.2"
   },
   "peerDependencies": {
-    "@shopgate/pwa-common": "^6.2.1",
-    "@shopgate/pwa-common-commerce": "^6.2.1",
-    "@shopgate/pwa-ui-shared": "^6.2.1"
+    "@shopgate/pwa-common": "^6.5.4",
```

Comment:
is it really needed to to increase the min required PWA version here?
We should keep the min version as low as possible

Reply:
Reverted to ^6.2.1